schemas/vlan.yml

```diff
@@ -0,0 +1,33 @@
+- name: vlan
+  title: VLAN
+  group: 2
+  short: Fields to describe observer vlan information.
+  description: >
+    The VLAN fields describe information about ingress and egress vlans reported 
+    by an observer.
```

Comment:
I think we should use the `vlan` description to give some guidance on when to use `network.vlan.*` vs `network.inner|outer.vlan.*` e.g. when populating inner/outer, should the user also populate `network.vlan.*`? If so, with which value?

Same thing with `observer.ingress|egress.vlan.*` vs `network*.vlan.*`. Should they be populated at the same time? If not, when to pick one vs the other?

Reply:
good ideas - I'll put something in

Reply:
I'm going to update vlan.yml to be a little more prescriptive, both in the specific vlan fields, as well as utilizing them for network state (vlan of traffic, inner, outer, etc.

Reply:
Thanks for the adjustments so far. The `observer.*` vs `network.*` vlan fields usage is clearer now.

I'm not sure we've clarified whether `network.vlan.*` needs to be populated, when `network.inner|outer.vlan.*` are populated. When we populate inner/outer VLAN info, should we also populate `network.vlan.*`? If so, with what?

Reply:
well thats a bugger now that you have me thinking about it... maybe we can come to a quick decision on the call